sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscribeProject.groovy

```diff
@@ -75,12 +75,17 @@ class SubscribeProject implements SubscribeProjectInput {
                 if (!subscriber.lastName) throw new ValidationException(
                         "Please provide a first name.")
         }
+        private static Consumer<Subscriber> validateTitle = {
+            Subscriber subscriber ->
+                if (!subscriber.title) throw new ValidationException(
+                        "Please provide a title.")
+        }
```

Comment:
I am unsure, why an application that is responsible for showing sample status information and enables status notification subscription has actual logic for person data validation. 

We have the offer manager application, where we include the person master data management, and I would like to keep it contained in this single app, to honor the SRP also on architecture level. If we start to introduce logic into to many applications, we will not be able to keep that in sync.

Reply:
This does not validate person data but subscription data. As we do not validate it anywhere else this has to be in here.